tests/testthat/test-derive_vars_dy.R

```diff
@@ -1,5 +1,5 @@
-#----test 1----
-  test_that("Test1::Single --DT input when ref date is --DTM", {
+# derive_vars_dy Test 1----
+  test_that("derive_vars_dy Test1: Single --DT input when ref date is --DTM", {
     datain <- tibble::tribble(
```

Comment:
I was thinking this Test 1---- stuff would get remove as we don't do it in the other testing files

Reply:
I think it is the other way around. We should add it to all other testing files. As this is a lot of work, we want to investigate if it could be automated (see https://github.com/pharmaverse/admiral/issues/714).